pipelines/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 resources:
 - docker-build
 - docker-build-oci-ta
+- docker-build-multi-arch-oci-ta
```

Comment:
We have both `multi-arch` and `multi-platform`. Let's stick to one of them.

Reply:
I will change to multi-platform. The remaining use of the term `multi-arch` in the repository comes from the `show-sbom` task.